engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/util/TimeCharacteristicCompatibility.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.engine.flink.util
+
+import com.github.ghik.silencer.silent
+import org.apache.flink.streaming.api.TimeCharacteristic
+import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
+import pl.touk.nussknacker.engine.flink.api.timestampwatermark.TimestampWatermarkHandler
+
+import scala.annotation.nowarn
+
+//Setting time characteristic is not needed and deprecated in Flink >= 1.12. We use this class not to pollute codebase
```

Comment:
I wonder if we can write some info how this change will evolve? What will happen when  `env.setStreamTimeCharacteristic` will be removed from flink? Do we remove those `useEventTime` invocations and lost compatibility with older versions?

Reply:
I added comment. Flink maintains deprecated methods for quite a long time